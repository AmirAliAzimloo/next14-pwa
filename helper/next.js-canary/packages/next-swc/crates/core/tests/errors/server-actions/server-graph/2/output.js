/* __next_internal_action_entry_do_not_use__ {"ac840dcaf5e8197cb02b7f3a43c119b7a770b272":"bar"} */ import { createActionProxy } from "private-next-rsc-action-proxy";
import { encryptActionBoundArgs, decryptActionBoundArgs } from "private-next-rsc-action-encryption";
'use strict';
export function bar() {}
import { ensureServerEntryExports } from "private-next-rsc-action-validate";
ensureServerEntryExports([
    bar
]);
createActionProxy("ac840dcaf5e8197cb02b7f3a43c119b7a770b272", bar);
