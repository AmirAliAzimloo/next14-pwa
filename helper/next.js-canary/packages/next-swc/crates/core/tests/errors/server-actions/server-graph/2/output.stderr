
  x Server actions must be async functions
   ,-[input.js:6:1]
 6 | 
 7 | export function bar() {}
   :                 ^^^
   `----
